### ISSUE TITLE
CommitAnalyze a prefix which shouldn't publish

### ISSUE THREAD

**Issue by evenstensberg** [semantic-release/commit-analyzer#76]
Is there any way to don't publish when a certain prefix matches, such as `docs`? Instead of publishing a new patch for each version, it would be nice to publish when code actually changes, not the docs.

```js
"analyzeCommits": {
      "preset": "angular",
      "releaseRules": [
        {
          "type": "docs",
          "release": "none"
        },
   }
```

**Comment by gr2m** [comment]
that is the current behavior, commits with the "docs" prefix don’t trigger a release.

If it did for you, can you point to the repository where we can see that happening?

**Comment by evenstensberg** [comment]
Was just wondering if I set release to none, it won't publish and errors won't be thrown instead of throwing an error at the release script

**Comment by pvdlg** [comment]
If you don't set any `releaseRules` and use the default (i.e. you don't set `releaseRules` at all) the `docs` commits will not be published. And you shouldn't have any errors.

If you have an error or a `docs` commit triggers a release, then the problem is somewhere else.
We can't help you to analyze that unless you provide us more info such as:
- The repo you are using
- The CI logs
- Your current config
- The problematic commits

**Comment by evenstensberg** [comment]
Gotcha. Thanks for elaborating!
